src/vscode-bicep/package.json

```diff
@@ -808,6 +808,7 @@
     "@azure/arm-resources": "^5.2.0",
     "@azure/arm-subscriptions": "^5.1.0",
     "@azure/identity": "^4.5.0",
+    "@microsoft/vscode-azext-azureauth": "^3.0.0",
```

Comment:
BTW I ran into the same pipeline failures you're getting after updating packages. I was able to fix it by updating the package lock by running `npm i --install-links` (as opposed to just `npm i`).

Reply:
Yep, thanks.  I actually have a .batch file in vscode-bicep to handle this properly (plus building the UI library):  npm-install.bat and npm-install.sh.  Unfortunately I discovered my .sh version was missing the --install-links thus the issue. :-)

Maybe a checkin test to verify...

Reply:
Hopefully we won't be needing `npm i --install-links` in a couple of weeks...I'm moving the `visualizer` and the `deployment pane` app to `vscode-bicep-ui`, which should address the issue.